deployment/src/main/java/io/quarkiverse/jasperreports/deployment/RepositoryProcessor.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkiverse.jasperreports.deployment;
+
+import io.quarkiverse.jasperreports.repository.DefaultReadOnlyRepository;
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildStep;
+
```

Comment:
This could probably just be in JasperProcessor as it's basically to support Jasper?

Reply:
Sure, just didn't want to overload it.